Firmware/ESP32/Hardware_bringup/sdkconfig

```diff
@@ -300,19 +301,19 @@ CONFIG_ADC_CAL_LUT_ENABLE=y
 #
 # Common ESP-related
 #
+# CONFIG_ESP_TIMER_PROFILING is not set
 CONFIG_ESP_ERR_TO_NAME_LOOKUP=y
 CONFIG_ESP_SYSTEM_EVENT_QUEUE_SIZE=32
 CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=2304
 CONFIG_ESP_MAIN_TASK_STACK_SIZE=3584
 CONFIG_ESP_IPC_TASK_STACK_SIZE=1024
 CONFIG_ESP_IPC_USES_CALLERS_PRIORITY=y
+CONFIG_ESP_TIMER_TASK_STACK_SIZE=3584
```

Comment:
add comment on default stack size. Also, whats the reason that we increased stack?